docs/examples/evaluation/multi_modal/multi_modal_rag_evaluation.ipynb

```diff
@@ -0,0 +1,1242 @@
+{
```

Comment:
<p>can we do if/else instead of commenting out. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8852/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>yeah that's a nice idea</p>